include/_kernel.h

```diff
@@ -209,18 +163,7 @@
 
 /****************************************************************************/
 
-/* before 3.9 all qualifiers are mapped to image write type */
-#if ((__clang_major__ < 4) && (__clang_minor__ < 9))
-
-#undef CLANG_HAS_IMAGE_AS
-#define IMG_WO_AQ
-#define IMG_RO_AQ
-#define IMG_RW_AQ
-
-#else
-
-/* 3.9 supports both read and write access qualifiers */
-
+// TODO this is always defined
```

Comment:
Address or remove this TODO?

Reply:
At the very least pls change the comment to /* */ ;)

Reply:
Yep this i forgot, will address